Automattic/jetpack
# Sharing Block: ensure color from block takes over color from theme

## Proposed changes:

See https://github.com/Automattic/jetpack/pull/36181#pullrequestreview-1922733824

This fixes a problem with some classic themes that define a specific color for links in post content. If you insert the sharing block in that post content, it then inherits the link color from the theme instead of getting the color we've defined for sharing buttons.

This PR adds a more specific rule to work around that.

### Other information:


## Jetpack product discussion

* N/A

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

* Install and activate the Twenty Ten theme
* Go to Posts > Add New
* In that new post, add a sharing buttons block, and add a button like Telegram.
* View the post on the frontend.
    * The button should not be blue.
